services/bwdserver-deployment/shipit.yaml

```diff
@@ -7,7 +7,7 @@ k8s:
 
     # If you change the nginx override, it needs a rolling restart
     custom-post-apply:
-      - kubectl rollout restart --namespace default deployment bwdserver-deployment
```

Comment:
how did this work before? Does `kubectl` just assume the `/`?

Reply:
yeah